gh-repo-topic

```diff
@@ -35,7 +35,7 @@ add() {
     usage >&2
     return 1
   fi
-  printf '{"names":[%s]}' "$(list | encode_topics "$@")" | \
+  printf '{"names":[%s]}' "$(list | awk NF | encode_topics "$@")" | \
```

Comment:
TIL `awk NF`! This solution works, however since the bug is in `encode_topics` not handling "empty" starting topics, I would rather move the solution there. In the `while read -r topic` loop there could be a condition:
```sh
[ -n "$topic" ] || continue
```